packages/tabs/src/Tabs.tsx

```diff
@@ -234,6 +234,7 @@ function Tabs({
     const { disabled, onClick } = child.props;
 
     const tabProps = {
+      ...child.props,
```

Comment:
a few things:
- we should preserve `child.props.onKeyDown`
- we should preserve `child.props.className`
- spread should go last so the child can overwrite `as` and `darkMode` if they wish. not sure whether it makes sense for `disabled` to be overwritable